packages/@react-spectrum/meter/docs/Meter.mdx

```diff
@@ -0,0 +1,116 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/meter';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Flex} from '@react-spectrum/layout';
+import {Meter} from '@react-spectrum/meter';
+```
+
+# Meter
+
+<p>{docs.exports.Meter.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Meter']}
+  sourceData={[{type: 'Spectrum', url: 'https://spectrum.adobe.com/page/meter/'}]} />
+
+## Example
+
+```tsx example
+<Meter label="Storage space" variant="positive" value={35} />
+```
+
+## Value
+
+Meters are controlled with the `value` prop representing current percentage (as an integer).
+In this case, the minimum and maximum values default to 0 and 100, respectively.
+
+```tsx example
+<Meter label="Storage space" value={25} variant="positive" />
+```
+
+Alternatively, a different scale can be used by setting the `minValue` and `maxValue`.
+
+```tsx example
+<Meter label="Widgets Used" minValue={50} maxValue={150} value={100} variant="positive" />
+```
+
+Values are formatted as a percentage by default, but can be modified by using the `formatOptions` for different formats.
```

Comment:
This is probably not a docs question, but I noticed we support `formatOptions` and the example is currency, which matches `ProgressBar`, but that doesn't match the use cases from the spectrum docs. It seems like an example with label of "1 to 4" would better match their intention. My real question is, should meter support `formatOptions` or should that only be a `ProgressBar` feature. I assume they both inherit from the same base and that is why `Meter` does, but should it?

Reply:
i defer to @dannify

Reply:
Meter could be used for things like a fund raising goal. So it should probably support all of the formatting